app/gateway/database/postgres/transfers/create_transfers.go

```diff
@@ -0,0 +1,28 @@
+package postgres_transfer
+
+import (
+	"context"
+	"stoneBanking/app/domain/entities/transfer"
+)
+
+func (repository transferRepository) Create(ctx context.Context, transferData transfer.Transfer) (transfer.Transfer, error) {
+	var sqlQuery = `
+	INSERT INTO
+			transfers (id,external_id, account_origin_id, account_destiny_id, ammount, created_at)
+	VALUES
+			($1, $2, $3, $4, $5, $6)
+	`
+	_, err := repository.db.Exec(
+		sqlQuery,
+		transferData.ID,
+		transferData.External_ID,
+		transferData.Account_origin_id,
+		transferData.Account_destination_id,
+		transferData.Amount,
+		transferData.Created_at)
+
+	if err != nil {
+		return transfer.Transfer{}, err
```

Comment:
É interessante sempre que for retornar um erro tentar adicionar alguma informação sobre o contexto que causou o erro. Isso ajuda(e muito) a identificar a causa de eventuais problemas. Aqui poderia ficar algo assim:
```go
return transfer.Transfer{}, fmt.Errorf("failed to transfer %d from %s to %s, %w", transferData.Amount, transferData.Account_origin_id, transferData.Account_destination_id, err)
```

Reply:
Rafael, nesse caso o err do banco já costuma trazer todas essas informações, o que eu pretendo em caso de erro jogar no log, para documentar os dados direitinho, e sanitizando a informação no retorno para a api externa.